compose/install/compose-linux.md

```diff
@@ -0,0 +1,120 @@
+---
+description: How to install Docker Compose on Linux
+keywords: compose, orchestration, install, installation, docker, documentation
+toc_max: 3
+
+title: Install on Linux
+redirect_from:
+- /compose/compose-plugin/
+---
+
+On this page you can find instructions on how to install the Compose on Linux from the command line.
+
+## Install Compose
+
+To install Compose:
+* Option 1: [Set up Docker's repository on your Linux system](#install-using-the-repository).
+* Option 2: [Install Compose manually](#install-the-plugin-manually).
+
+### Install using the repository
+
+> **Note**
+>
+> These instructions assume you already have Docker Engine and Docker CLI installed and now want to install the Compose plugin.  
+For Compose standalone, see [Install Compose Standalone](./compose-other.md#on-linux).
+
+If you have already set up the Docker repository, jump to step 2.
+
+1. Set up the repository in:
+[Ubuntu](../../engine/install/ubuntu.md/#set-up-the-repository) |
+[CentOS](../../engine/install/centos.md/#set-up-the-repository) |
+[Debian](../../engine/install/fedora.md/#set-up-the-repository) |
+[RHEL](../../engine/install/fedora.md/#set-up-the-repository) |
+[SLES](../../engine/install/sles.md/#set-up-the-repository).
+
+2. Update the `apt` package index, and install the _latest version_ of Docker Compose:
+
+    > Or, if using a different distro, use the equivalent package manager instructions.
+
+    ```console
+    $ sudo apt-get update
+    $ sudo apt-get install docker-compose-plugin
+    ```
+
+    Alternatively, to install a specific version of the Compose CLI plugin:
+
+    a. List the versions available in your repo:
+
+      ```console
+      $ apt-cache madison docker-compose-plugin
+
+      docker-compose-plugin | 2.6.0~ubuntu-bionic | https://download.docker.com/linux/ubuntu bionic/stable amd64 Packages
+      docker-compose-plugin | 2.5.0~ubuntu-bionic | https://download.docker.com/linux/ubuntu bionic/stable amd64 Packages
+      docker-compose-plugin | 2.3.3~ubuntu-bionic | https://download.docker.com/linux/ubuntu bionic/stable amd64 Packages
+      ```
+
+    b. From the list obtained use the version string you can in the second column to specify the version you wish to install.
+
+    c. Install the selected version:
+
+
+      ```console
+      $ sudo apt-get install docker-compose-plugin=<VERSION_STRING>
+      ```
+    where `<VERSION_STRING>` is, for example,`2.3.3~ubuntu-focal`.
+
+3.  Verify that Docker Compose is installed correctly by checking the version.
+
+    ```console
+    $ docker compose version
+    Docker Compose version v2.3.3
+    ```
+
+> **Note**
+>
+> To run Compose as a non-root user, see [Manage Docker as a non-root user](../../engine/install/linux-postinstall.md){:target="_blank" rel="noopener" class="_"}.
```

Comment:
This is a general topic for using `docker` as a non-root user; as the page starts with the assumption users already have a functional docker installation, this wouldn't be needed for most of those cases. That said, perhaps we should include this as a link in an "next steps" / "where to go next" section at the bottom of the page.

Reply:
Good point. Adding this to the batch of changed, I'll prompt you for another review when ready.

Reply:
Suggestion to link to this topic http://localhost:4000/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user.